wandb/cli/cli.py

```diff
@@ -1680,6 +1680,16 @@ def launch(
     hidden=True,
     help="a wandb client registration URL, this is generated in the UI",
 )
+@click.option(
+    "--verbose", "-v", is_flag=True, default=False, help="Display verbose output"
+)
+@click.option(
+    "--very-verbose",
+    "-vv",
+    is_flag=True,
+    default=False,
+    help="Display verbose output, including logging output",
+)
```

Comment:
https://click.palletsprojects.com/en/8.1.x/options/#counting why not just use verbose counting type i think this is more standard

Reply:
That's cool! Definitely gonna do that instead